lib/cfn-model/model/api_gateway_stage.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require_relative 'model_element'
+
+class AWS::ApiGateway::Stage  < ModelElement
+  attr_accessor :usage_plans, :deployment_id
```

Comment:
usage_plan is singular?

Reply:
can deployment_id be multiple?

Reply:
@erickascic  Yea usage_plan should be singular, will make that change. And the property for `DeploymentId` on the Stage resource only accepts a single value.